src/w3d/renderer/dx8caps.cpp

```diff
@@ -366,12 +379,14 @@ bool DX8Caps::Is_Valid_Display_Format(int w, int h, WW3DFormat format)
 void DX8Caps::Check_Texture_Format_Support(WW3DFormat format, const w3dcaps_t &caps)
 {
     if (format == WW3D_FORMAT_UNKNOWN) {
-        memset(m_supportTextureFormat, 0, sizeof(m_supportTextureFormat));
+        memset(m_supportTextureFormat, false, sizeof(m_supportTextureFormat));
```

Comment:
Actually 0 is correct. memset expects integer.

If you really want to pass bool, then you can use `std::fill`.